mitmproxy/models/http.py

```diff
@@ -226,9 +225,8 @@ class HTTPResponse(MessageMixin, Response):
 
         headers: Headers object
 
-        content: Content of the request, None, or CONTENT_MISSING if there
-        is content associated, but not present. CONTENT_MISSING evaluates
-        to False to make checking for the presence of content natural.
+        content: Content of the request, the value is None if there is content
```

Comment:
Ah yes - it's ok.
For some reason the Github diff view indicated a different class...
<img width="671" alt="screen shot 2016-03-26 at 18 07 15" src="https://cloud.githubusercontent.com/assets/114300/14061316/aef166d2-f37d-11e5-9207-78e09ae204d8.png">